packages/block-serialization-default-parser/parser.php

```diff
@@ -48,11 +48,26 @@ class WP_Block_Parser_Block {
 	 */
 	public $innerHTML;
 
-	function __construct( $name, $attrs, $innerBlocks, $innerHTML ) {
+	/**
+	 * List of string fragments and null markers where inner blocks were found
+	 *
+	 * @example array(
+	 *   'innerHTML'    => 'BeforeInnerAfter',
+	 *   'innerBlocks'  => array( block, block ),
+	 *   'innerContent' => array( 'Before', null, 'Inner', null, 'After' ),
+	 * )
+	 *
+	 * @since 4.2.0
+	 * @var array
+	 */
+	public $innerContent;
+
+	function __construct( $name, $attrs, $innerBlocks, $innerHTML, $innerContent ) {
 		$this->blockName   = $name;
 		$this->attrs       = $attrs;
 		$this->innerBlocks = $innerBlocks;
 		$this->innerHTML   = $innerHTML;
+		$this->innerContent = $innerContent;
```

Comment:
New assignment introduces misalignment on the `=`.

Reply:
will follow-up in a new PR, thanks for spotting this!